src/unix/linux_like/linux/mod.rs

```diff
@@ -1319,6 +1319,11 @@ s! {
         pub propagation: crate::__u64,
         pub userns_fd: crate::__u64,
     }
+
+    // is __be16 __bitwise __u16
+    pub struct __c_anonymous___be16 {
```

Comment:
As above re: __c_anonymous - it also might be helpful just to have a type like this higher up in the hierarchy to be shared, I find the idea that only Linux/Android have bigendian u16s unlikely.